organization/philanthropy.md

```diff
@@ -0,0 +1,34 @@
+---
+type: resource
+explains: our policy on philanthropy
+---
+
+# Philanthropy policy
+
+The Foundation for Public Code is currently wholly supported by philanthropists who share our commitment to [our mission](https://about.publiccode.net/organization/mission.html). All donations are unearmarked and go into our general operating budget, in compliance with our status as a Dutch [public benefit organization](https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/business-public-benefit-organisations/public_benefit_organisations/public_benefit_organisations) (ANBI).
+
+We will be philanthropically funded until we have enough members paying dues to support a sustainable blended financial model.
```

Comment:
The goal here is not to have a blended model, it is to make sure all sustainable operations can be payed for with public means to reduce the influence of fundraising activities. At that point philantropy becomes and engine for growth without it being able to influence our existing responsibilities.

Reply:
We discussed this at great length. We agree that that's the goal, but thought the financial model page was a better place for this information. The "blended model" was a short way of signalling that we expect to need both member dues and philanthropy to operatate for the foreseeable future. 
Do you think it's important to put this future goal on our philanthropy policy page?

Reply:
I created a separate issue to update the financial model: #787

Reply:
I think the main goal of this page would be to indicate that we don't let our fundraising impact how we try to achieve our mission, not through influence, having donors add requirements to their donation, or availability, changing our practices to get more money.

To me the notion of a 'blended' model signals that we plan on becoming a fundraising NGO, where we're trying to become public infrastructure.

Perhaps something like 'We will be philantropically funded to support growth and a larger impact while we try the operational core of delivering on our responsibilities from our members, guaranteeing the public purpose.' or something less abstract ;)

Reply:
So, to answer the question, yes, I think it's important, but the details should definetely go into the financial model :)

Reply:
This was how I defined it in an early version of the case for support:

The Foundation for Public Code is primarily member-funded, with additional philanthropic support. Core stewardship operations and member relations are covered by membership dues to make sure we’re sustainable in the long term.

We welcome/embrace philanthropy for:
+ awareness raising
+ecosystem growth
+ in-depth education
+ toolset improvement

This lets us try out new ideas, expand our service offering and reach potential new members and partners without compromising our core activities.

Reply:
@bvhme Check now?